project.clj

```diff
@@ -62,7 +62,11 @@
                  ;; in different versions of the three different logback artifacts
                  [net.logstash.logback/logstash-logback-encoder]
 
-                 [puppetlabs/jruby-utils "0.8.0"]
+                 [puppetlabs/jruby-utils "0.8.1-SNAPSHOT"
```

Comment:
This PR will fail Travis for now b/c of a dependency on work being done in a [related jruby-utils PR](https://github.com/puppetlabs/jruby-utils/pull/45) which will eventually need to be pulled in for this PR.